docs/release-notes/2.41.0.md

```diff
@@ -20,29 +20,6 @@ Percona Monitoring and Management (PMM) is an open source database monitoring, m
 
 ## Release Highlights
 
-### Offline metric collection
-
-The offline metric collection introduced in v2.33.0 has now been extended to QAN metrics. Previously, it was limited to Prometheus exporters only.
-
-PMM data is crucial for monitoring and troubleshooting. PMM operates on a client-server architecture, exchanging data between clients (on database servers) and servers (PMM UI interface and data storage). 
-
-Previously, if the PMM client-server connection was interrupted by a network or application issue, the system would stop collecting data, resulting in data collection interruption.
-
-Starting with PMM 2.41.0, if the PMM client and server components encounter connection problems, the QAN data gathered during these periods is stored and transmitted immediately after the connection is restored.
-
-
-### Technical Preview of PMM in HA mode
```

Comment:
Hi.  I do not get why we are removing the section `Technical preview of PMM in HA mode` from the `Release Highlights`.  I thought we wanted to remove the section `Offline metric collection` (related to PMM-8655) and that we wanted to keep the technical preview of the HA mode (as we have documented it in the how-to folder).  Can this be clarified, please.  I think the section `Technical preview of PMM in HA mode` should not be removed.  Thank you

Reply:
I have confirmed it with @rnovikovP and he does not want me to include it.